### PR TITLE
Stop using untyped dicts to represent quotes

### PR DESCRIPTION
**Pull request by Weves**

### CODE REVIEW

File: backend/danswer/direct_qa/on_startup.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+            return True
+        except AuthenticationError:
+            return False
+        except Exception as e:
+            logger.warning(f"GenAI API key failed for the following reason: {e}")
+
+    return False
+
+
+def get_default_backend_qa_model(
```
Comment (yuhongsun96) [inline backend/danswer/direct_qa/on_startup.py]: Let's call this get_default_llm <- can reuse for rephrasing questions etc.

File: backend/danswer/direct_qa/on_startup.py [hunk 2]
```diff
@@ -0,0 +1,111 @@
+from typing import Any
```
Comment (yuhongsun96) [inline backend/danswer/direct_qa/on_startup.py]: how about llm_utils.py

**Review by yuhongsun96 — APPROVED** [review]
Nit stuff only, ideally address

File: backend/danswer/direct_qa/on_startup.py (hunk 1 above)
Comment (Weves) [reply backend/danswer/direct_qa/on_startup.py]: Done!

File: backend/danswer/direct_qa/on_startup.py (hunk 2 above)
Comment (Weves) [reply backend/danswer/direct_qa/on_startup.py]: Seems good

**Pull request merged**